text/0004-channel-participation-api-without-system-channel.md

```diff
@@ -231,38 +235,56 @@ or
 
 or
 
 ```GET /channels/channel-name```
 
-The response will be successful if the channel exists, and fail otherwise. If successful, it will return the
-informational object described above.
+The response will be successful if the channel exists, and fail otherwise. If successful, it will return a more detailed
+JSON informational object that includes:
+- the channel name 
+- a relative resource URL that identifies the channel
+- the cluster-relation of the channel: whether it is a `member` or a `follower`
+- status of the channel: whether it is `onboarding` (catching-up towards the join-block) or `active` (past the join-block)
+- ledger height
+
+In case an OSN is joined as a follower, invoking "List channel-name" conveys the information of whether this OSN had 
+finished catching up with the cluster. This can be the sign for the channel admin to add the OSN to the channel's 
+consenters set. When the OSN is a member of the cluster, following the height can help detect connectivity and 
+configuration problems. 
+
+This informational object is included in the response to a successful "Join".  
 
-- In case an OSN is joined as a follower, invoking "List channel-name"   
-  conveys the information of whether this OSN had finished catching up with the cluster.
-- When the OSN is a member of the cluster, following the height can help detect connectivity and configuration problems.
+The `List` command works also _when the system channel exists_, and on any consensus type. Therefore, to support that:
+- In an `etcdraft` OSN:
+  - If the channel is serviced by this OSN, cluster-relation is `member` and status is `active`; and 
+  - If the channel is not serviced by this OSN, cluster-relation is `config-tracker` and status is `inactive`.
+- In a `kafka` or `solo` OSN, cluster-relation is `none` and the status is `active`.
 
 ## Security
 The channel participation API adopts the style of the
 ["Operations" API]((https://hyperledger-fabric.readthedocs.io/en/release-2.0/operations_service.html#operations-security))
-in the orderer. The local orderer channel participation admin is not identified by the MSP, but rather through mutual TLS.
-Like in the operations API that handles log-spec, the orderer.yaml file specifies:
+in the orderer. The _local orderer channel participation admin_ is not identified by the MSP, but rather through mutual TLS.
+Like in the operations API that handles log-spec, the `orderer.yaml` file specifies:
 - whether the channel participation API is enabled
 - whether security is enabled for channel participation
+- a listening address (host:port)
+- the server certificate and private key
 - the client root CAs that are authorized to invoke channel participation commands
 
+In order to separate the roles of "operations admin" and "local orderer channel participation admin" we intend to expose a 
+new endpoint (host:port) that is dedicated to the channel participation API. This means that the orderer configuration, as 
```

Comment:
I would say rather than 'dedicated to the channel participation API', maybe 'dedicated to Fabric administrative tasks, like the channel participation API'.  We could for instance extend this endpoint with other query operations.

Reply:
done